test/APINotes/Inputs/custom-frameworks/APINotesFrameworkTest.framework/Headers/APINotesFrameworkTest.apinotes

```diff
@@ -39,6 +39,27 @@ SwiftVersions:
   - Version: 3.0
     Classes:
       - Name: TestProperties
+        Methods:
+          - Selector: accessorsOnlyRenamedRetyped
+            MethodKind: Instance
+            SwiftName: 'renamedAndRetyped()'
+            ResultType: 'id _Nonnull'
```

Comment:
We don't even really need API notes in the test case. The accessors could just be declared explicitly with mismatched types, and everything should Just Work™.